tests/utils/helper/helper.go

```diff
@@ -65,6 +65,19 @@ func DigArray(object interface{}, keys ...interface{}) []interface{} {
 	return result
 }
 
+func DigArrayToString(object interface{}, keys ...interface{}) []string {
+	value := Dig(object, keys)
+	//ExpectWithOffset(1, value).ToNot(BeNil())
```

Comment:
lots of commented-out code. can we get rid of those throughout this PR?

Reply:
Good point, will remove the commented lines